assets/configure

```diff
@@ -3301,7 +3301,7 @@ fi
 
 # Define the identity of the package.
  PACKAGE='tango'
- VERSION='9.2.5'
+ VERSION='@VERSION@'
```

Comment:
As I already wrote in another comment, we should not modify the configure script.
We should rather change configure.ac and use the boostrap script to generate a fresh configure script with the autotools.